designs/jane/src/body.mjs

```diff
@@ -0,0 +1,227 @@
+
+    points.neckLeft = points.middle.shiftTowards(points.leftShoulder, neckWidth / 2)
+    points.neckRight = points.middle.shiftTowards(points.rightShoulder, neckWidth / 2)
+
+    const neckDepthFront = measurements.hpsToBust * options.neckDepthFront
+
+    points.middleHem = points.bottomLeft.shiftTowards(points.bottomRight, widthBody / 2)
+
+    points.neckFront = points.middle
+      .shiftTowards(points.middleHem, neckDepthFront)
+      .attr('data-text', 'Front Neckline')
+      .attr('data-text-class', 'center')
+
+    points.neckCp1 = points.neckRight.shift(90, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckCp2 = points.neckFront.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+
+    paths.neckLine = new Path()
+      .move(points.neckRight)
+      .curve(points.neckCp1, points.neckCp2, points.neckFront)
+
+    points.neckCp3 = points.neckLeft.shift(90, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckCp4 = points.neckFront.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+
+    paths.neckLine2 = new Path()
+      .move(points.neckFront)
+      .curve(points.neckCp4, points.neckCp3, points.neckLeft)
+
+    const neckDepthBack = measurements.hpsToBust * options.neckDepthBack
+
+    points.neckBack = points.middle
+      .shiftTowards(points.neckFront, neckDepthBack)
+      .attr('data-text', 'Back Neckline')
+      .attr('data-text-class', 'center')
+
+    points.neckBackCp1 = points.neckBack.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckBackCp2 = points.neckBack.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckBackCp3 = points.neckLeft.shift(90, points.neckBack.dy(points.neckLeft) * 0.8)
+    points.neckBackCp4 = points.neckRight.shift(90, points.neckBack.dy(points.neckRight) * 0.8)
+
+    paths.NecklineBack1 = new Path()
+      .move(points.neckRight)
+      .curve(points.neckBackCp4, points.neckBackCp2, points.neckBack)
+
+    paths.necklineBack2 = new Path()
+      .move(points.neckBack)
+      .curve(points.neckBackCp1, points.neckBackCp3, points.neckLeft)
+
+    points.topTopLeft = points.topLeft.shift(90, 20)
+    points.topTopRight = points.topRight.shift(90, 20)
+    paths.extraBoundary = new Path()
+      .move(points.topLeft)
+      .move(points.topRight)
+      .move(points.topTopRight)
+      .move(points.topTopLeft)
+      .move(points.topLeft)
+      .close()
+
+    // Complete?
+    if (complete) {
+      points.logo = points.middle.shiftTowards(points.middleHem, lengthBody / 4)
+      snippets.logo = new Snippet('logo', points.logo)
+
+      points.title = points.logo.shiftTowards(points.middle, lengthBody / 12)
+      macro('title', {
+        at: points.title,
+        nr: 1,
+        title: 'body',
+      })
+
+      if (sa) {
+        paths.sa = paths.sideseam.offset(sa).setClass('fabric sa')
+      }
+
+      macro('cutonfold', {
+        from: points.topLeft,
+        to: points.topRight,
+        grainline: true,
+      })
```

Comment:
It really depends on the linen you're using. Since the pattern is supposed to be close to zero waste, the width of the body and the width of the fabric would inform whether you cut it cross grain or lengthwise. So maybe the pattern shouldn't have a grainline at all.

Reply:
Than shouldn't grainline be false either way?

Reply:
Yeah you're right!